### PR TITLE
Replace mailing-list with slack in Introduction

### PR DESCRIPTION
**Pull request by sypets**
Replace information about mailing-list with information about slack channel in Introduction

### CODE REVIEW

File: Documentation/Introduction/Index.rst [hunk 1]
```diff
@@ -56,29 +57,31 @@ to the "examples" extension at some later stage.
...
-^^^^^^^^
-
-For general questions about the documentation get in touch by writing
-to `documentation@typo3.org <mailto:documentation@typo3.org>`_ .
+How to contribute
+=================
 
 If you find a bug in this manual, please be so kind as to check the
-online version on https://docs.typo3.org/typo3cms/CoreApiReference/.
+online version on http://docs.typo3.org/typo3cms/InsideTypo3Reference/.
```
Comment (marble) [inline Documentation/Introduction/Index.rst]: InsideTypo3?

File: Documentation/Introduction/Index.rst [hunk 2]
```diff
@@ -56,29 +57,31 @@ to the "examples" extension at some later stage.
...
 From there you can hit the "Edit me on GitHub" button in the top right corner
-and submit a pull request via GitHub. Alternatively you can just file an issue
-using the bug tracker: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues.
+and submit a pull request via GitHub. Alternatively you can just `file an issue
+using the bug tracker <https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-InsideTypo3/issues>`__.
```
Comment (marble) [inline Documentation/Introduction/Index.rst]: InsideTypo3?

**Comment by sypets** [comment]
whoops. I used the text from "Inside TYPO3". I can fix this tomorrow.

**Comment by lolli42** [comment]
I just merged a similar patch that overworked the Introduction chapter a bit and solved the 'slack' issue along the way: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/commit/d828067900a540fa773af84f8f73e056d1dd172a
I think this PR can be closed.

**Pull request closed without merging**